actions/mongodb_explain_action.go

```diff
@@ -0,0 +1,75 @@
+// pmm-agent
+// Copyright 2019 Percona LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package actions
+
+import (
+	"context"
+	"strings"
+
+	"github.com/percona/percona-toolkit/src/go/mongolib/explain"
```

Comment:
We should not add that dependency (and an old MongoDB driver) just for https://github.com/percona/percona-toolkit/blob/3.0/src/go/mongolib/explain/explain.go

Reply:
Removed